devtools/conda-envs/test_env.yaml

```diff
@@ -0,0 +1,30 @@
+name: test
+channels:
+  - conda-forge
+  - omnia
+
+dependencies:
+    # Base depends
+  - python
+  - pip
+
+    # Testing
+  - pytest
+  - pytest-cov
+  - nbval
+  - codecov
+  - numpy
+  - networkx
+  - parmed
+  - packaging
+  - openmmtools
+  - pyyaml
+  - toml
+  - bson
+  - msgpack-python
+  - xmltodict
+
+    # Pip-only installs
+  - pip:
+    - coverage==4.4
```

Comment:
Coverage is now on conda and conda-forge.

Reply:
Moved to conda installs in 9c47611